include/seastar/net/tls.hh

```diff
@@ -152,6 +152,10 @@ namespace tls {
      */
     using dn_callback = noncopyable_function<void(session_type type, sstring subject, sstring issuer)>;
 
+    class certificate_credentials;
+    void* get_impl(certificate_credentials const&);
```

Comment:
In the interest of upstreaming this, can we split this into two commits; one for the callback changes, and another for this `get_impl`.

Reply:
👍